.changeset/quick-rocks-float.md

```diff
@@ -0,0 +1,6 @@
+---
+'@modern-js/runtime': patch
+---
+
+feat: export getMonitors for server component
```

Comment:
Is this changeset correct ?

Reply:
Yeah，The PR does two things
![image](https://github.com/user-attachments/assets/5876bd30-1833-4ef1-892b-566c37796cf7)